CONTRIBUTING.md

```diff
@@ -0,0 +1,618 @@
+
+The token and pid data will be hold by the new Process structure and
+they are related to a container.
+
+Fixes #123
+
+Signed-off-by: Ragni Garg <ragni.garg@in.ibm.com>
+```
+
+The issue is automatically closed by GitHub when the
+[commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) is parsed.
+
+## GitHub workflow
+
+Raksh employs certain augmentations to a 
+[standard GitHub workflow](https://guides.github.com/introduction/flow/). 
+In this section, we explain these augmentations in more detail. Follow these guidelines when contributing to Kata Containers repositories, except where noted below.
+
+* Complete the [GitHub basic setup](#github-basic-setup) above before continuing.
+
+* Ensure each PR only covers one topic. If you mix up different items in
+  your patches or PR, they will likely need to be reworked.
+
+* Follow a [topic branch method](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches) for development.
+
+* Follow carefully the [patch format](#patch-format) for PRs.
+
+* Apply the appropriate GitHub labels to your PR. This
+  is particularly relevant to maintain [Stable branch backports](#stable-branch-backports). See also [GitHub labels and keywords that block PRs](#github-labels-and-keywords-that-block-prs)
+
+  >**Note:** External contributors should use keywords, explained in the
+  >link above. Labels may not be visible to external contributors. 
+
+* [Rebase](https://help.github.com/en/github/using-git/about-git-rebase)
+  commits on your branch and `force push` after each cycle of feedback.
+
+#### Configure your environment
+
+Most [Raksh repository](https://github.com/IBM/raksh)
+contain code written in the [Go language (golang)](https://golang.org/). Go 
+requires all code to be put inside the directory specified by the `$GOPATH` 
+variable. Follow this example to put the code in the standard location.
+
+```sh
+$ export GOPATH=${GOPATH:-$HOME/go}
+$ mkdir -p "$GOPATH"
+```
+
+>*Note*: If you intend to make minor edits, it's acceptable
+> to simply fork and clone without adding the GOPATH variable.
+
+##### Fork and clone
+
+In this example, we configure a Git environment to contribute to this very 
+`Community` repo. We create a sample branch, incorporate reviewer feedback, and rebase our commits.
+
+1. Fork the [upstream repository](https://help.github.com/articles/cloning-a-repository):
+
+1. [Clone your forked copy of the upstream repository](https://help.github.com/articles/cloning-a-repository):
+
+1. While on your *forked copy*, select the green button `Clone or download`
+   and copy the URL. 
+
+1. Run the commands below and **paste the copied URL** (previous step),
+   so your real GitHub user name replaces `your-github-username` below.
+  
+```sh
+$ mkdir -p $GOPATH/src/github.com/ibm
+$ cd $GOPATH/src/github.com/ibm
+$ git clone https://github.com/ibm/raksh.git
+$ git checkout -b 1.9.1-raksh origin/1.9.1-raksh
```

Comment:
This should refer to the forked copy
$ git clone https://github.com/<github-username>/raksh
$ cd raksh